### PR TITLE
[glacierui] lockscreen style change

### PR DESCRIPTION
**Pull request by Morpog**

### CODE REVIEW

File: src/qml/LockscreenClock.qml [hunk 1]
```diff
@@ -3,21 +3,18 @@ import QtQuick 2.0
...
     gradient: Gradient {
-        GradientStop { position: 0.0; color: '#dd000000' }
-        GradientStop { position: 0.5; color: '#aa000000' }
+        GradientStop { position: 0.0; color: '#a0000000' }
         GradientStop { position: 1.0; color: '#00000000' }
     }
 
     Column {
         id: clockColumn
+        spacing: -15
```
Comment (faenil) [inline src/qml/LockscreenClock.qml]: ?? :)

Comment (Morpog) [inline src/qml/LockscreenClock.qml]: fixed in next commit :D

File: src/qml/LockscreenClock.qml [hunk 2]
```diff
@@ -44,18 +39,17 @@ Rectangle {
...
+            opacity: 0.8
+            horizontalAlignment: Text.AlignHCenter
 
             anchors {
                 left: parent.left
                 right: parent.right
             }
 
-            text: Qt.formatDateTime(wallClock.time, "dd/MM/yyyy")
+            text: Qt.formatDateTime(wallClock.time, "<b>dddd</b>, dd. MMMM yyyy")
```
Comment (sledges) [inline src/qml/LockscreenClock.qml]: Please change to as per mockup (most region-neutral format): `<b>dddd</b>, d MMMM yyyy`
Mental note: we'll need to add locale handling here one day :)

**Comment by sledges** [comment]
LGTM, boom!

**Pull request merged**